app/validators/idv/in_person/residential_address_validator.rb

```diff
@@ -0,0 +1,20 @@
+module Idv
+  module InPerson
+    module ResidentialAddressValidator
```

Comment:
Real talk, this is not dry.  I'm not sure if there's a way to extend the `Idv::FormAddressValidator` for these slightly differently named fields.  Feedback would be appreciated, please!

Reply:
i see we include that validator in the `Idv::InPerson::FormAddressValidator` since the 1 difference btwn them is the added check for radio btn value. I feel like since the field names are different it should be ok for us to add this additional file. But yeh it does feel weird since they seem so similar.